index.html

```diff
@@ -5,9 +5,29 @@
     <meta http-equiv="X-UA-Compatible" content="IE=edge">
     <meta name="viewport" content="width=device-width, initial-scale=1.0">
     <title>Mohamed Saleh Portfolio</title>
+    <link rel="preconnect" href="https://fonts.googleapis.com">
+    <link rel="preconnect" href="https://fonts.gstatic.com" crossorigin>
+    <link href="https://fonts.googleapis.com/css2?family=Inter:wght@100;200;300;400;500;600;700;800;900&family=Lato:wght@300&display=swap" rel="stylesheet">
     <link rel="stylesheet" href="css/main.css">
 </head>
 <body>
-    
+    <header class="container">
+        <p>Welcome</p>
+        <nav>
+            <img src="images/Union.png" alt="menu-icon">
+        </nav>
+    </header>
+    <main>
+        <div class="section-1">
+            <p class="p1 container">Hey there. I’m Mohamed <span>I’m a software developer</span></p>
+            <p class="p2 container"> I can help you build a product , feature or website Look through some of my work and experience! If you like what you see and have a project you need coded, don’t hestiate to contact me.</p>
+            <div class="icons container">
+                <a href="https://twitter.com/mohamed20163858"><img src="images/twiter.png" alt="twitter-icon"></a>
+                <a href="https://www.linkedin.com/in/mohamed-mohsen-685a4995/"><img src="images/linkedinIcon.png" alt="Linkedin-icon"></a>
+                <a href="https://github.com/mohamed20163858"><img src="images/github.png" alt="github-icon"></a>
+                <a href="https://angel.co/u/mohamed-mohsen-4"><img src="images/angellist.png" alt="angellist-icon"></a>
+            </div>
```

Comment:
This content should be aligned inside a `<ul>` with each icon wrapped in an `<li>`. Please fix it.

<img width="604" alt="Screen Shot 2022-06-29 at 20 03 48" src="https://user-images.githubusercontent.com/82426682/176570925-45707582-99dc-4223-8cc2-ff268adeabf4.png">